### DISCUSSION TITLE
I would love to actually try the app so much, but..

### DISCUSSION THREAD

**Discussion by AS0007** [flxzt/rnote#689] [Q&A]
I get that version 6.0 is unstable, and it's literally unusable, crashes a lot and lags with everything, completely horrible, but the thing is i tried to install version 5.8 but i can't bec. there's no .exe file(I am windows 10) and the RAR says that the folder is corrupt, i tried extracting it anyways but it got to a certain point and then displayed an error, how can i get my hands on a stable version? i see a lot of potential in this app but i just can't try or do anything. I promise i tried to search for this on google and reddit but absolutely nothing showed due the app unfortunately being very unpopular. TIA

**Comment by PhilDevProg** [comment]
Windows support was only added in the last version. But the last version shouldn't be that unstable that it doesn't work... Could you please file a bug report?
